wd_helper.au3

```diff
@@ -1027,19 +1028,52 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ;                  - $_WD_ERROR_InvalidDataType
 ;                  - $_WD_ERROR_InvalidExpression
 ;                  - $_WD_ERROR_InvalidArgue
+;                  - $_WD_ERROR_ElementIssue
+;                  - $_WD_ERROR_NoMatch
 ; Author ........: Danp2
 ; Modified ......: mLipok
-; Remarks .......: If no option is selected, SELECTEDINDEX will return -1
+; Remarks .......: If no option is selected, SELECTEDINDEX will return -1.
 ; Related .......: _WD_FindElement, _WD_ExecuteScript
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand, $aParameters = Default)
+Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand, $vParameters = Default)
 	Local Const $sFuncName = "_WD_ElementSelectAction"
+	Local Const $sParameters = 'Parameters:    Command = ' & $sCommand &  '    Parameters = ' & ((IsArray($vParameters)) ? ("(array)") : ("(string)"))
 	Local $vResult, $sScript
-
 	Local $sNodeName = _WD_ElementAction($sSession, $sSelectElement, 'property', 'nodeName')
 	Local $iErr = @error, $iExt = 0
+	Local Static $sScript_MultiSelectTemplate = StringReplace( _ ; it is declared as static to optimize AutoIt processing speed - this line will be processed once per script run
+			"function MultiSelectOption(SelectElement, LabelsToSelect, AllowMultiple) {" & _
+			"	if (AllowMultiple && SelectElement.multiple == false) {" & _
+			"		return '';" & _
+			"	};" & _
+			"	var options = SelectElement.options;" & _
+			"	var result = false;" & _
+			"	for (var i = 0, o; i < options.length; i++) {" & _
+			"		o = options[i];" & _
+			"		if (		(LabelsToSelect.indexOf(o.label)!= -1)" & _
+			"				&&	(o.disabled == false	&& (!(o.parentNode.nodeName == 'OPTGROUP' && o.parentNode.disabled)))" & _
+			"				&&	(o.hidden == false	&& (!(o.parentNode.nodeName == 'OPTGROUP' && o.parentNode.hidden)))" & _
+			"			) {" & _
+			"				if (AllowMultiple == false) {" & _
+			"					SelectElement.selectedIndex = -1;" & _
+			"				};" & _
+			"				o.selected = true;" & _
+			"				result = true;" & _
+			"			};" & _
```

Comment:
This section of code could be revised so that a `break;` occurs to exit the loop once the correct option is found when AllowMultiple is False

Reply:
Please take a look on: [break + CleanUp](https://github.com/Danp2/au3WebDriver/pull/367/commits/538d86395eb251f70473cc869f013e895397af5c)